### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kbergin**
Added content from the doc we were working on

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,48 @@
...
+The Job Manager is an API and UI for monitoring and managing jobs in a backend execution engine. 
+
+The Broad, Verily, and many other organizations in the life sciences execute enormous numbers of scientific workflows and need to manage those operations. Job Manager was born out of the experiences of producing data for some of the world’s largest sequencing projects such as The Cancer Genome Atlas, Baseline, and the Thousand Genomes Project. 
+
+The Job Manager aspires to bring ease and efficiency to developing and debugging workflows while seamlessly scaling to production operations management.
+
+## Key Features
+* Sits on top of either [Cromwell](https://github.com/broadinstitute/cromwell) or [dsub](https://github.com/googlegenomics/dsub)
+* Service provider interface can be extended to support other engines
+* Rich, search capabilities across current and historic workflows
```
Comment (calbach) [inline README.md]: Rich, search -> Rich search

File: README.md [hunk 2]
```diff
@@ -1,6 +1,48 @@
...
+The UI must be deployed along with a backend implementation of the API; two such implementations are provided here:
+
+### Cromwell
+Monitors jobs launched by the [Cromwell workflow engine](https://github.com/broadinstitute/cromwell). The Python Flask wrapper was created using Swagger Codegen and can be configured to pull data from a specific Cromwell instance. The Job Manager currently supports Cromwell version 29.
+
+### dsub
+
+Monitors jobs that were launched via the [dsub](https://github.com/googlegenomics/dsub) CLI. Thin stateless wrapper around the dsub Python library. Authorization is required for deploying the UI, which is used to communicate with the [Google Genomics Pipelines API](https://cloud.google.com/genomics/pipelines). The wrapper itself is implemented in Python Flask using Swagger codegen models.
+
+Note that a “task” in dsub nomenclature corresponds to a Job Manager API’s “job”. A Dockerfile is provided which serves for production deployment using gunicorn.
```
Comment (calbach) [inline README.md]: Now that I'm rereading what I wrote, perhaps move the second sentence up to the end of the previous paragraph.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,48 @@
...
+## What is the Job Manager?
+
+The Job Manager is an API and UI for monitoring and managing jobs in a backend execution engine. 
+
+The Broad, Verily, and many other organizations in the life sciences execute enormous numbers of scientific workflows and need to manage those operations. Job Manager was born out of the experiences of producing data for some of the world’s largest sequencing projects such as The Cancer Genome Atlas, Baseline, and the Thousand Genomes Project. 
+
+The Job Manager aspires to bring ease and efficiency to developing and debugging workflows while seamlessly scaling to production operations management.
+
+## Key Features
+* Sits on top of either [Cromwell](https://github.com/broadinstitute/cromwell) or [dsub](https://github.com/googlegenomics/dsub)
```
Comment (calbach) [inline README.md]: nit: maybe adjust the wording slightly - current "either" seems too exclusive of other possible backends, maybe "Supports job visualization over Cromwell or dsub backends"?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,48 @@
 # Job Manager
+[Development Readme](#development)
```
Comment (calbach) [inline README.md]: See also the [development guide] below

File: README.md [hunk 5]
```diff
@@ -1,6 +1,48 @@
...
 
-Job Manager API and UI for interacting with asynchronous batch jobs.
+## What is the Job Manager?
```
Comment (calbach) [inline README.md]: Do we need this header?

**Comment by kbergin** [comment]
Thanks @calbach , added in your suggestions!

File: README.md [hunk 6]
```diff
@@ -1,6 +1,46 @@
 # Job Manager
+See also the [development guide] below(#development)
```
Comment (rexwangcc) [inline README.md]: Maybe we could use either `[development guide below](#development)` or `[development guide](#development) below` to make the anchor link works.

File: README.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 # Job Manager
-See also the [development guide](#development) below
+See also the [development guide](#development) below.
```
Comment (rexwangcc) [inline README.md]: This looks better. Didn't notice that the dot is blocking the anchor from working lol Thanks for the change here!

File: README.md (hunk 6 above)
Comment (kbergin) [reply README.md]: No don't worry, it wasn't the period, it was because I didn't keep the () near the [] part in a very silly move!! Good catch here!

**Pull request merged**